docs/basic-features/data-fetching/get-static-props.md

```diff
@@ -74,6 +74,20 @@ This means that instead of fetching an **API route** from `getStaticProps` (that
 
 Take the following example. An API route is used to fetch some data from a CMS. That API route is then called directly from `getStaticProps`. This produces an additional call, reducing performance. Instead, the logic for fetching the data from the CMS can be moved to `getStaticProps`.
 
+```jsx
+// pages/page.js
+
+// This function runs only on the server side
+export async function getStaticProps() {
+  // Call an external API endpoint to get posts
+  const res = await fetch('https://.../posts/')
```

Comment:
Instead of a fetch here should we make this more clear that the logic can be shared by using a lib instead? e.g.
```js
// pages/blog.js
import loadPosts from '../lib/load-posts'

export async function getStaticProps() {
  // Instead of fetching our `/api` route we can call the same
  // function directly in `getStaticProps`
  const posts = await loadPosts()
  
  // Props returned will be passed to the page component
  return { props: { posts } }
}
```

Reply:
I think that's a great idea! Thanks, @ijjk, I will update this PR.